config.ts

```diff
@@ -266,6 +266,7 @@ const siteConfig = {
   footer: {
     newsletter: {
       text: 'Stay up to date with the latest features and changes to Prisma',
+      BREVO_API_KEY: process.env.BREVO_API_KEY,
```

Comment:
I submitted a new email address and nothing happen. Aren't missing the API KEY here?